src/test/java/com/rybka/repository/CurrencyHistoryRepositoryTest.java

```diff
@@ -57,17 +55,41 @@ public void testOnCorrectHistoryFindTop5() {
         var historyOtherList = List.of(
                 buildRandomHistory(), buildRandomHistory(), buildRandomHistory()
         );
-        repository.saveAll(historyOtherList);
-        repository.saveAll(historyTop5List);
+
+        historyOtherList
+                .forEach(history -> {
+                    try {
+                        Thread.sleep(1000);
```

Comment:
What the purpose of Thread.sleep? 
Can't you just save as list?

Reply:
fixed